crates/typst/src/layout/grid/mod.rs

```diff
@@ -701,15 +701,15 @@ pub struct GridVLine {
 ///
 /// #grid(
 ///   fill: (x, y) => rgb(
-///     if calc.odd(x + y) { "EFF0F3" }
-///     else { "7F8396" }
+///     if calc.odd(x + y) { "7F8396" }
+///     else { "EFF0F3" }
 ///   ),
 ///   columns: (1em,) * 8,
 ///   rows: 1em,
 ///   align: center + horizon,
 ///
 ///   [♜], [♞], [♝], [♛], [♚], [♝], [♞], [♜],
-///   [♟], [♟], [♟], [♟], [],  [♟], [♟], [♟],
+///   [♟], [♟], [♟], [♟], [],   [♟], [♟], [♟],
```

Comment:
This seems to have undone the alignment in the source code.

```suggestion
///   [♟], [♟], [♟], [♟], [],  [♟], [♟], [♟],
```

Reply:
Oooh I see. This is because chess pieces are not displayed with a monospace font on my end.